lohica-js-network/src/actions/logginActions.js

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
```

Comment:
rename file logginActions.js to loginAction.js

Reply:
done